docker-compose.yml

```diff
@@ -119,5 +119,6 @@ services:
       REACT_APP_EOS_API_PROTOCOL: "${REACT_APP_EOS_API_PROTOCOL}"
       REACT_APP_EOS_CHAIN_ID: "${REACT_APP_EOS_CHAIN_ID}"
       REACT_APP_BLOCK_EXPLORER: "${REACT_APP_BLOCK_EXPLORER}"
+      REACT_APP_NETWORK_MONITOR_URL: "${REACT_APP_NETWORK_MONITOR_URL}"
```

Comment:
Add this to variable to `docker-compose.staging.yml` please

Reply:
This file need more atention, i create new issue for this topic https://github.com/eoscostarica/eos-rate/projects/1#card-65477072